test/e2e/packetcapture_test.go

```diff
@@ -694,6 +776,85 @@ func getOSString() string {
 	return "linux"
 }
 
+func getIPsForEndpoint(t *testing.T, data *TestData, podRef *crdv1alpha1.PodReference, ipStr *string) *PodIPs {
```

Comment:
I assume you are using the `PodIPs` return type for convenience here, even though that's a bit misleading as the IPs don't have to belong to a Pod?

Reply:
I guess in the context of the e2e tests, it is actually always Pod IPs, as we always use Pods for the source / destination, even when explicitly providing an IP address instead of an actual Pod reference...